src/textdescriptives/components/descriptive_stats.py

```diff
@@ -94,18 +113,22 @@ def syllables(self, doc: Doc) -> dict:
             dict: syllables_per_token_mean, syllables_per_token_median,
                 syllables_per_token_std
         """
-        n_syllables = doc._._n_syllables
-        if not n_syllables:
+        nan_output = {
+            "syllables_per_token_mean": np.nan,
+            "syllables_per_token_median": np.nan,
+            "syllables_per_token_std": np.nan,
```

Comment:
slightly more bug-resistant would be to define the variables and then only return the dict once (otherwise you might change the name "syllables_per_token_mean" one place but not the other.